api/portainer.go

```diff
@@ -844,6 +843,7 @@ type (
 	JWTService interface {
 		GenerateToken(data *TokenData) (string, error)
 		ParseAndVerifyToken(token string) (*TokenData, error)
+		SetUserSessionDuration(userSessionDuration string) error
```

Comment:
I believe we should pass a time.Duration parameter here as we're already parsing the string duration outside of the Setter (in the Validate of the API operation). This would also change the signature to a proper Setter without returning any error.